idb/postgres/postgres_integration_test.go

```diff
@@ -0,0 +1,136 @@
+package postgres
+
+import (
+	"database/sql"
+	"fmt"
+	"testing"
+
+	_ "github.com/lib/pq"
+	"github.com/orlangure/gnomock"
```

Comment:
I still don't like it but I'll let it go since it works on travis. Next time I try to run tests I'll have to find a way to hack it out and make it optional and disabled if `docker` isn't available.

Reply:
I'm sure we can figure something out, the connection string environment var should work.